resources/views/cruds/datagrids/filters/_array.blade.php

```diff
@@ -11,15 +11,29 @@
         ) !!}
     </div>
     <div class="col-xs-4">
-        {!! Form::select(
-            $field['field'] . '_option',
-            [
-                '' => __('crud.filters.options.include'),
-                'exclude' => __('crud.filters.options.exclude'),
-                'none' => __('crud.filters.options.none'),
-            ],
-            $filterService->single($field['field'] . '_option'), [
-                'class' => 'form-control entity-list-filter',
-        ]) !!}
+        @if (isset($field['withChildren']) && $field['withChildren'] === true )
+            {!! Form::select(
+                $field['field'] . '_option',
+                [
+                    '' => __('crud.filters.options.include'),
+                    'children' => __('crud.filters.options.children'),
```

Comment:
Lots of copy-paste here when the only difference is if children is included in the list ;-)